analytics/views/support.py

```diff
@@ -166,6 +168,7 @@ def support(
         default=None, str_validator=check_string_in(VALID_MODIFY_PLAN_METHODS)
     ),
     scrub_realm: bool = REQ(default=False, json_validator=check_bool),
+    delete_user_by_email: Optional[str] = REQ(default=None),
```

Comment:
Hmm, could we do `delete_user_by_id`? I like working with IDs because that generally feels safer / more precise.

Reply:
Sure, reworked it. I thought a bit about whether that doesn't undermine the "type the email address to confirm" prompt, but I think it doesn't and it's fine?

Added a bonus commit to fix our case-sensitive user lookup for that endpoint, which can be pretty annoying sometimes.

Reply:
Ahh, good call.